test/memory.test.js

```diff
@@ -430,6 +431,17 @@ describe('Memory connector', function() {
       });
     });
 
+    it('should search array for value', function(done) {
+      User.find({where: {favoriteNumbers: 1}}, function(err, users) {
```

Comment:
I think we probably should introduce a different operator such as `contains` instead of overloading the default operator (eq).